index.html

```diff
@@ -176,5 +176,33 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section id="form-section">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h2>
+
+      <form class="" action="index.html" method="post">
+        <div class="fullname">
```

Comment:
- [ ] According to the project requirement you should use the [formspree service](https://formspree.io/html).[formspree service](https://formspree.io/html) that uses the `action` and `method` attributes. 
`* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`